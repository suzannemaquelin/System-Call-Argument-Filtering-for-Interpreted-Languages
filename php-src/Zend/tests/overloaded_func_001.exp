Fatal error: Uncaught Error: Cannot call overloaded function for non-object in %soverloaded_func_001.php:%d
Stack trace:
#0 {main}
  thrown in %soverloaded_func_001.php on line %d