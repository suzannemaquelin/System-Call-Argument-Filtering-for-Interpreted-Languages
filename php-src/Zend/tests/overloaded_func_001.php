<?php
var_dump(_ZendTestClass::test());
?>
