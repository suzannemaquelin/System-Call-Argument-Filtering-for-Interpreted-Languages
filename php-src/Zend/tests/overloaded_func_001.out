Failed loading /opt/php-7.1/lib/php/extensions/debug-non-zts-20160303/xdebug.so:  /opt/php-7.1/lib/php/extensions/debug-non-zts-20160303/xdebug.so: cannot open shared object file: No such file or directory

Fatal error: Uncaught Error: Cannot call overloaded function for non-object in /root/php-src/Zend/tests/overloaded_func_001.php:2
Stack trace:
#0 {main}
  thrown in /root/php-src/Zend/tests/overloaded_func_001.php on line 2